src/KafkaFlow.UnitTests/OffsetCommitterTests.cs

```diff
@@ -4,6 +4,7 @@ namespace KafkaFlow.UnitTests
     using System.Collections.Generic;
     using System.Linq;
     using System.Threading;
+    using System.Threading.Tasks;
     using Confluent.Kafka;
     using FluentAssertions;
```

Comment:
Please consider removing unnecessary usings. Also [this one](https://github.com/Farfetch/kafkaflow/pull/409/files#diff-20a8b61d7d160af60b910dce35d003fbdbcc7242cb908fe3aa6c26bc53cd5823R6)

Reply:
done